packaging/aznfs/DEBIAN/prerm

```diff
@@ -24,6 +24,19 @@ if [ "$1" == 'remove' -o "$1" == 'purge' ]; then
 			exit 1
 		fi
 	fi
+
+	existing_mounts_for_nfsv4FilesShares=$(cat /opt/microsoft/aznfs/data/aznfs_files_mountmap 2>/dev/null | egrep '^\S+' | wc -l)
+	if [ $existing_mounts_for_nfsv4FilesShares -ne 0 ]; then
+		echo
+		echo -e "${RED}There are existing Azure Files NFS mounts using aznfs mount helper, they will not be tracked!"
+		echo -n -e "Are you sure you want to continue? [y/N]${NORMAL} "
+		read -n 1 result
+		echo
+		if [ "$result" != "y" -a "$result" != "Y" ]; then
+			echo "Removal aborted!"
+			exit 1
```

Comment:
same thing with the comment below. if turns out we need to change that one, then this one needs changes too.

Reply:
merged code for v3 and v4